Gopkg.toml

```diff
@@ -24,13 +24,18 @@ ignored = ["github.com/fnproject/fn/cli",
            "github.com/Microsoft/go-winio*",
            "github.com/Azure/go-ansiterm*"]
 
+[prune]
+  go-tests = true
+  unused-packages = true
+  non-go = true
```

Comment:
the other 2 seems fine but `non-go` we may be missing some assembler/c files (sqlite, at least?) we might like to have the code from for optimization, license files for legal reasons. while it's probably a good idea to run tests on all of our dependencies especially for patches like this one, it's not something we have been doing...

Reply:
License files are not removed...
87 are still in vendor dir. 👍

Reply:
thanks